frontend/src/index.scss

```diff
@@ -0,0 +1,18 @@
+@import "../node_modules/bulma/bulma.sass";
+@import "../node_modules/bulma-extensions/bulma-accordion/src/sass/index.sass";
+@import "../node_modules/bulma-extensions/bulma-badge/src/sass/index.sass";
+@import "../node_modules/bulma-extensions/bulma-calendar/src/sass/index.sass";
+@import "../node_modules/bulma-extensions/bulma-carousel/src/sass/index.sass";
+@import "../node_modules/bulma-extensions/bulma-checkradio/src/sass/index.sass";
+@import "../node_modules/bulma-extensions/bulma-divider/src/sass/index.sass";
+@import "../node_modules/bulma-extensions/bulma-iconpicker/src/sass/index.sass";
+@import "../node_modules/bulma-extensions/bulma-pageloader/src/sass/index.sass";
+@import "../node_modules/bulma-extensions/bulma-pricingtable/src/sass/index.sass";
+@import "../node_modules/bulma-extensions/bulma-quickview/src/sass/index.sass";
+@import "../node_modules/bulma-extensions/bulma-ribbon/src/sass/index.sass";
+@import "../node_modules/bulma-extensions/bulma-slider/src/sass/index.sass";
+@import "../node_modules/bulma-extensions/bulma-steps/src/sass/index.sass";
+@import "../node_modules/bulma-extensions/bulma-switch/src/sass/index.sass";
+@import "../node_modules/bulma-extensions/bulma-tagsinput/src/sass/index.sass";
+@import "../node_modules/bulma-extensions/bulma-timeline/src/sass/index.sass";
+@import "../node_modules/bulma-extensions/bulma-tooltip/src/sass/index.sass";
```

Comment:
Do you *really* need all extensions of Bulma right at the beginning of the project? ^^